great_expectations/expectations/regex_based_column_map_expectation.py

```diff
@@ -187,7 +187,7 @@ def validate_configuration(
     def _question_renderer(cls, configuration, result=None, runtime_configuration=None):
         column = configuration.kwargs.get("column")
         mostly = configuration.kwargs.get("mostly")
-        regex = getattr(cls, "regex")
+        regex = cls.regex
```

Comment:
```suggestion
        regex = getattr(cls, "regex", None)
```
Not sure about this one...